packages/caver-utils/src/utils.d.ts

```diff
@@ -0,0 +1,179 @@
+    isBloom(bloom: string): boolean
+    isTopic(topic: string): boolean
+    checkAddressChecksum(address: string): boolean
+    utf8ToHex(str: string): string
+    hexToUtf8(hex: string): string
+    hexToNumber(value: string | number | BN): string
+    hexToNumberString(value: string | number | BN): string
+    numberToHex(value: string | number | BN): string
+    toHex(value: string | number | BN | object, returnType?: boolean): string
+    bufferToHex(buf: Buffer | unknown[] | string | number | BN | object): Buffer
+    toBuffer(input: Buffer | unknown[] | string | number | BN | object): Buffer
+    numberToBuffer(num: number | string | BN): Buffer
+    hexToBytes(hex: string): number[]
+    bytesToHex(bytes: number[]): string
+    isHex(hex: string): boolean
+    isHexStrict(hex: string): boolean
+    leftPad(string: string, chars: number, sign?: string): string
+    rightPad(string: string, chars: number, sign?: string): string
+    toTwosComplement(number: number | string | BN): string
+    sha3(value: string | BN): string | null
+    parsePredefinedBlockNumber(blockNumber: string): string
+    isPredefinedBlockNumber(blockNumber: string): boolean
+    isValidBlockNumberCandidate(blockNumber: string | number): boolean
+    isValidPrivateKey(privateKey: string): boolean
+    isValidNSHSN(value: string | number): boolean
+    parsePrivateKey(
+        privateKey: string
+    ): {
+        privateKey: string
+        address: string
+        isHumanReadable: boolean
+    }
+    parseKlaytnWalletKey(key: string): string[]
+    isKlaytnWalletKey(privateKey: string): boolean
+    isContractDeployment(txObject: TransactionObject): boolean
+    rlpEncode(data: EncodeInput): string
+    rlpDecode(encodedData: string): string | string[]
+    xyPointFromPublicKey(pub: string): [string, string]
+    resolveSignature(signature: string | string[] | object): string[]
+    transformSignaturesToObject(signatures: string | string[] | SignatureData | SignatureObject): SignatureObject
+    transformSignaturesToObject(signatures: string[][] | SignatureData[] | SignatureObject[]): SignatureObject[]
+    transformSignaturesToObject(
+        signatures: string | string[] | string[][] | SignatureData | SignatureData[] | SignatureObject | SignatureObject[]
+    ): SignatureObject | SignatureObject[]
+    getTxTypeStringFromRawTransaction(rawTransaction: string): string
+    trimLeadingZero(hex: string): string
+    makeEven(hex: string): string
+    txTypeToString: string
+    isValidPublicKey(publicKey: string): boolean
+    isCompressedPublicKey(publicKey: string): boolean
+    compressPublicKey(uncompressedPublicKey: string): string
+    decompressPublicKey(compressedPublicKey: string): string
+    /** @deprecated */
+    isTxHash(txHash: string): boolean
+    /** @deprecated */
+    isTxHashStrict(txHash: string): boolean
+    isValidHash(hash: string): boolean
+    isValidHashStrict(hash: string): boolean
+    isValidRole(role: Role): boolean
```

Comment:
```suggestion
    isValidRole(role: string): boolean
```
This can be just string type.